command/CommandController.js

```diff
@@ -14,6 +14,14 @@ const database = new Database();
 const HostnameHelper = require('./HostnameHelper.js');
 const hostnameHelper = new HostnameHelper();
 
+var endInstructionsData = {
+  text:
+    'Your article is finished. ' +
+    'To listen to more articles say "Alexa, tell Scout to get titles"',
+  url: '',
```

Comment:
Do we need to SSML anything here to make it sound better?

Reply:
Let's add a field for the outro voice and have that be an env var that we can change.  That way we can play with the voices here if we want to.  One of the things we want to do is have it read in a different voice.

Reply:
SSML gets added when the synth is made. We could add some more SSML but I think it sounds fine (but I'm not a native English speaker so I may be biased).

Adding the outro voice field in the afternoon